site/pages/docs/ref/wb-data-json/wb-data-json-en.hbs

```diff
@@ -267,7 +269,7 @@
 <section>
 <h2 id="jsonObjToArr">Template - Transforming JSON object into an array</h2>
 
-<p>Since WET 4.0.27 it is now possible to iterate anything that is not an array to render content through a <code>&lt;template&gt;</code>. It simply get transformed into a array. There is a low risk of lost of information, only one specific use case. It's quite similar to the JSON-LD Expanded Form. In this transformation process, it assume the array key is an IRI. Then the will result include some property from JSON-LD such as <code>@id</code> and <code>@value</code>. The <code>@id</code> wont be overwritten, in those such case the array key will be discard after the transformation.</p>
+<p>Since WET 4.0.27 or without the <code>streamline</code> flag since WET 4.0.64 it is possible to iterate anything that is not an array to render content through a <code>&lt;template&gt;</code>. It simply get transformed into a array. There is a low risk of lost of information, only one specific use case. It's quite similar to the JSON-LD Expanded Form. In this transformation process, it assume the array key is an IRI. Then the will result include some property from JSON-LD such as <code>@id</code> and <code>@value</code>. The <code>@id</code> wont be overwritten, in those such case the array key will be discard after the transformation.</p>
```

Comment:
May be worth adding a link to the IRI definition for general audience to get the reference :)

Reply:
I added a link to wikipedia about what is an IRI

Reply:
And I did the same fix to the French page.